specs/protocol/safer-safes.md

```diff
@@ -0,0 +1,509 @@
+# Safe Contract Extensions V2
```

Comment:
To add: The Timelock Guard will need a module exclusion list so that txns that originate from the Liveness module don't get stuck behind the timelock guard.